### PR TITLE
Add more noteheads

### PR DESCRIPTION
**Pull request by lasconic**

### CODE REVIEW

File: mscore/menus.cpp [hunk 1]
```diff
@@ -594,6 +594,10 @@ Palette* MuseScore::newNoteHeadsPalette()
             // HEAD_BREVIS_ALT shows up only for brevis value
             if (i == int(NoteHead::Group::HEAD_BREVIS_ALT) )
                   sym = Note::noteHead(0, NoteHead::Group(i), NoteHead::Type::HEAD_BREVIS);
+            if (i == int(NoteHead::Group::HEAD_PITCHNAME) )
```
Comment (Jojo-Schmitz) [inline mscore/menus.cpp]: isn't this rather an `else if`?

File: mscore/menus.cpp [hunk 2]
```diff
@@ -594,6 +594,10 @@ Palette* MuseScore::newNoteHeadsPalette()
...
+                  sym = Note::noteHead(0, NoteHead::Group(i), NoteHead::Type::HEAD_QUARTER, int(Tpc::TPC_B));
+            if (i == int(NoteHead::Group::HEAD_PITCHNAME_GERMAN) )
```
Comment (Jojo-Schmitz) [inline mscore/menus.cpp]: can't/shouldn't this get added to the above `if` with an `||`

File: libmscore/note.cpp [hunk 3]
```diff
@@ -239,7 +238,9 @@ static NoteHeadName noteHeadGroupNames[] = {
       {"re",        QT_TRANSLATE_NOOP("noteheadnames", "Re") },
       {"ti",        QT_TRANSLATE_NOOP("noteheadnames", "Ti") },
       {"name-pitch",          QT_TRANSLATE_NOOP("noteheadnames", "Pitch name") },
-      {"name-pitch-german",   QT_TRANSLATE_NOOP("noteheadnames", "Pitch name (german)") }
+      {"name-pitch-german",   QT_TRANSLATE_NOOP("noteheadnames", "Pitch name (german)") },
```
Comment (Jojo-Schmitz) [inline libmscore/note.cpp]: I think this is spelled "German"

Comment (Jojo-Schmitz) [inline libmscore/note.cpp]: I think this is spelled "German"

**Comment by lasconic** [comment]
Thank for the feedback. I would also be interested in feedback about the actual feature.

**Comment by Jojo-Schmitz** [comment]
For that I'd need to see it working. Would require me to build with your PR, I can try that, later.

**Comment by Jojo-Schmitz** [comment]
Not sure what is going on, but when I try I get clefs as notheads for some pitches:
Tab clef for Eb
Small Tab clef for E
Bass (F) clef for C
Treble (G) clef for Db
Percussion clef for D#

File: libmscore/note.cpp [hunk 4]
```diff
@@ -106,20 +212,74 @@ struct NoteHeadName {
...
+      {"xcircle",        QT_TRANSLATE_NOOP("noteheadnames", "XCircle") },
+      {"withx",          QT_TRANSLATE_NOOP("noteheadnames", "With X") },
+      {"triangle-up",    QT_TRANSLATE_NOOP("noteheadnames", "Triangle Up") },
+      {"triangle-down",  QT_TRANSLATE_NOOP("noteheadnames", "Triangle Down") },
+      {"slashed1",       QT_TRANSLATE_NOOP("noteheadnames", "Slashed bottom left to top right") },
+      {"slashed2",       QT_TRANSLATE_NOOP("noteheadnames", "Slashed top left to bottom right") },
+      {"diamond",        QT_TRANSLATE_NOOP("noteheadnames", "Diamond") },
+      {"diamond-old",    QT_TRANSLATE_NOOP("noteheadnames", "Diamond (Old)") },
+      {"circled",        QT_TRANSLATE_NOOP("noteheadnames", "Circled") },
+      {"circled-large",  QT_TRANSLATE_NOOP("noteheadnames", "Circled") },
```
Comment (Jojo-Schmitz) [inline libmscore/note.cpp]: Twice the same user visible and translatable text on different heads? Maybe better use `Circled (large)`?

**Comment by Jojo-Schmitz** [comment]
Ah, now I see, this happens with Emmentaler and Emmental only

**Comment by lasconic** [comment]
MScore font needs tweaking indeed. It should look ok with bravura for now.

**Comment by Jojo-Schmitz** [comment]
Yes, it does, as I found out seconds before yor reply. Looks OK in Gonvile and MuseJazz too

**Comment by lasconic** [comment]
Yes, they don't have the notehead... MScore somehow has old glyphs at the wrong place.

**Comment by lasconic** [comment]
@mgavioli Did you work on the MScore font? I could take a look this weekend if not but it's hard to merge changes on the font....

**Comment by Jojo-Schmitz** [comment]
Triange up/down look identical in Emmentaler (but wrong stem, too long, on one of them), different (and right) in Bravura

**Comment by Jojo-Schmitz** [comment]
Some moveble do glyphs have a similar issue with Emmentaler as the (other) notenames heads

**Comment by Jojo-Schmitz** [comment]
Seems 64672e9 fixes the issues with Emmentaler

**Comment by lasconic** [comment]
Indeed

**Comment by Jojo-Schmitz** [comment]
Hmmm, while the list in Inspector was to much of an exposure, having hidden it in the staff's advanced style properties is too little exposure to my taste.

**Comment by IsaacWeiss** [comment]
@lasconic It may be too late to mention this, but I heartily agree with @Jojo-Schmitz:
<blockquote>having hidden it in the staff's advanced style properties is too little exposure to my taste.</blockquote>

**Comment by lasconic** [comment]
Never too late to speak up. Jojo mentioned on IRC that we could add it in the instrument dialog together with standard since it's not used. Honestly, I still disagree with you two. We can't surface all features for all users. Some features need to remain "too little exposed". I believe this is one. We will see how the users react. Regarding "standard", maybe it should be removed since it's not used.

**Comment by Jojo-Schmitz** [comment]
The column is there in the instruments dialog and used for Tab (and maybe other staff types?), so why not just making use of it for a 'normal' staff, to allow for these new note heads styles more easily.
Also why is that button "Add to Templates" disabled? What is it good for, if not to create and save a custom staff template?

**Comment by Jojo-Schmitz** [comment]
Ah, I see why:
`void EditStaffType::addToTemplatesClicked()
      {
      qDebug("not implemented: add to templates");
      }`

**Comment by IsaacWeiss** [comment]
I see this as analogous to the existing Tools > Toggle Rythmic Slash Notation, and it seems strange that too fundamentally similar features should be accessed so differently.

**Comment by Jojo-Schmitz** [comment]
We probably better continue to discuss this in https://musescore.org/en/node/141551

**Pull request merged**